pkg/components/util/test/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 The Lokomotive Authors
```

Comment:
We should discourage using `pkg/components/util` package. I would put it somewhere in `internal`, so it doesn't get exported outside of `lokomotive` module.